naujas/app/code/Controller/Inbox.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Controller;
+
+use Core\AbstractController;
+use Core\Interfaces\ControllerInterface;
+use Helper\FormHelper;
+use Helper\Messages;
+use Helper\Url;
+use Model\Message;
+use Model\User;
+
+class Inbox extends AbstractController implements ControllerInterface
+{
+    public function __construct() {
+        parent::__construct();
+        if(!$this->isUserLogged()) {
+            Url::redirect("user/login");
+        }
+    }
+    public function index()
+    {
+        $messages = Message::getMessages($_SESSION["user_id"], "received");
+
+        if($messages) {
+            foreach ($messages as $message) {
+                $message->setSeen(1);
```

Comment:
Uzkrauni zinuciu lista ir jau seen? seen  manau turetu buti po to, kai ja atsidarai